docs/agents.md

```diff
@@ -323,27 +323,27 @@ sequenceDiagram
     Note over Agent: Game session complete
 ```
 
-### Retrievers, tools, and schema
+### tools and schema
 
-Under the hood, retrievers use the model's "tools" or "functions" API to let the model know what retrievers are available to call. Tools or functions are also used to define the schema(s) for structured responses, thus a model might have access to many tools, some of which call retrievers while others end the run and return a result.
+Under the hood, tools use the model's "tools" or "functions" API to let the model know what tools are available to call. Tools or functions are also used to define the schema(s) for structured responses, thus a model might have access to many tools, some of which call tools while others end the run and return a result.
```

Comment:
This sentence needs a bit of updating because it was previously using the word "tools". Not sure the best way to differentiate between LLM-provider tools and pydantic-ai tools in terminology, but I'm hoping me just pointing this out is enough to be useful. 😄